keras/engine/topology.py

```diff
@@ -1220,6 +1220,7 @@ def call(self, inputs, mask=None):
             l2 = inputs[1]
             denominator = K.sqrt(K.batch_dot(l1, l1, self.dot_axes) *
                                  K.batch_dot(l2, l2, self.dot_axes))
+            denominator = K.maximum(denominator, 1e-15)
```

Comment:
Please use instead `K.epsilon()`, so that the user can globally set all fuzz factors throughout the codebase.